This condition makes it link to wrong library on Gentoo Prefix on Debian
--- a/components/CMakeLists.txt
+++ b/components/CMakeLists.txt
@@ -246,11 +246,7 @@ target_link_libraries(components
     RecastNavigation::Recast
     )
 
-if (BULLET_USE_DOUBLES AND (UBUNTU_FOUND OR DEBIAN_FOUND) AND OPENMW_USE_SYSTEM_BULLET)
-    target_link_libraries(components BulletCollision-float64 LinearMath-float64)
-else()
     target_link_libraries(components ${BULLET_LIBRARIES})
-endif()
 
 if (WIN32)
     target_link_libraries(components
